librad/src/net/quic/connection/tracking.rs

```diff
@@ -240,13 +238,28 @@ fn spawn_gc(
                 None => break,
                 Some(epoch) => {
                     let prev_epoch = epoch.fetch_add(1, SeqCst);
+                    let curr_epoch = epoch.load(SeqCst);
```

Comment:
You know why I did this? Because I **appreciate** vertical alignment! But then, automatic code formatters were invented ... :crying_cat_face: